hazelcast/src/main/java/com/hazelcast/util/executor/StripedExecutor.java

```diff
@@ -149,7 +149,15 @@ private Worker getWorker(Runnable command) {
         return workers[index];
     }
 
-    // used in tests.
+    public List<BlockingQueue<Runnable>> getWorkQueues() {
```

Comment:
Are you going to print all queued events? So what happens when there is for example 100k events pending? Will this string contain 100k events? Not very usable and dangerous to the diagnostics system because it will create huge strings and will quickly eat through the diagnostics files. It could potentially cause OOME's